Makefile

```diff
@@ -185,13 +185,11 @@ set-node-os-linux:
 	@kubectl label nodes --all kubernetes.io/os=linux --overwrite
 
 .PHONY: deploy-es-operator
-deploy-es-operator: set-node-os-linux set-max-map-count
+deploy-es-operator: set-node-os-linux set-max-map-count deploy-prometheus-operator
```

Comment:
The current code expects some CRDs to exist: `PROMETHEUS_RULES_CRD, `PROMETHEUS_SERVICE_MONITORS_CRD`. 

@kevinearls is this target used by CI jobs? If so, it means that the tests might be affected, probably by an extra delay and potentially a (small) overhead.

Reply:
CI jobs will use this if running on k8s.  On OCP they install the ES operator via OLM.

Reply:
Only during the ES job, right? In the case of this PR, it would be this run here: https://github.com/jaegertracing/jaeger-operator/pull/1157/checks?check_run_id=981180597

I think we are good then.

Reply:
@objectiser do you think it's OK to merge, or would you prefer keeping the current code, adding the extra Prometheus operator target independent from ES's requirements?

Reply:
:thumbsup:

Reply:
@jpkrohling Think it is ok to merge - forgotten that es-operator was dependent upon it.